pull_requests.md

```diff
@@ -28,15 +28,44 @@ its own reward. It will make you faster at solving your own problems, and if you
 you’ll find that bad documentation will never stand in your way again (ironically, stackoverflow will become less useful for languages where you have access to
 all of the source).
 
-## Readability and Maintainability
+In summary, the code review process serves the following purposes:
+* __knowledge transfer__
+  * team members learn tricks and best practices from each other
+  * better use of experts: 5-10% of their time spent reviewing code can make a meaningful contribution to a team
+  * within a few months of serious code reviews, every team member writes code on the level of the best developer
+* __better team work__
+  * ensuring agreed upon coding practices are followed
+  * team develops common code style while reviewing each others code
+  * better collaboration of remote teams
+  * more people stay up to date with whats going on in other parts of the code base → they can fix bugs there
+* __better code quality__
+  * team develops culture of high code quality (people are less sloppy if somebody else is looking)
+  * Identify issues in code that are not found through automated testing: poor code structure/architecture, performance bottlenecks, bad variable naming, etc
+  * Find and eliminate defects in the code early (when they are cheap to fix)
+* __feedback for developers__
+  * how good good/bad/average is my code?
+  * where am I in comparison to my peers?
+  * what is possible?
+  * how do others solve interesting problems?
+
+
+## Guidelines for people submitting a code review
+
+* Limit the total amount of code for a review to a single logical change.
+  Submit several pull requests for independent changes.
+  [Split up your feature branch](http://blog.originate.com/blog/2015/02/13/refactoring-git-branches-part-ii/) if necessary.
+* No more than 200–400 lines of code per PR
```

Comment:
I don't really like this, I think it could scare some junior devs who don't understand the spirit of what we're getting at. 400 lines of CoffeeScript is not the same as 400 lines of JavaScript is not 400 lines of HTML.